_stbt/core.py

```diff
@@ -511,17 +511,20 @@ class MotionResult(object):
     * ``motion``: Boolean result, the same as evaluating `MotionResult` as a
       bool. That is, ``if result:`` will behave the same as
       ``if result.motion:``.
+    * ``region``: The region of the video frame that contained the motion.
+      `None` if no motion detected.
     """
-    def __init__(self, timestamp, motion):
+    def __init__(self, timestamp, motion, region):
         self.timestamp = timestamp
         self.motion = motion
+        self.region = region
```

Comment:
I wasn't too worried about this break, but I suppose defaulting to `None` doesn't hurt.